lib/devise_otp_authenticatable/controllers/helpers.rb

```diff
@@ -27,7 +27,7 @@ def otp_t()
       end
 
 
-      def trusted_devices_enabled?
+      def trusted_devices_enabled?(resource=resource)
```

Comment:
`resource=resource` looks funny... maybe choose a different name for the parameter?